python/sglang/srt/cpu_utils.py

```diff
@@ -49,3 +58,14 @@ def reset_param_data_if_needed(param_data, dim, start, length):
 
     param_data.narrow(dim, start, length).zero_()
     return
+
+
+def cpu_has_amx_support():
+    return torch._C._cpu._is_amx_tile_supported() and is_intel_amx_backend_available
+
+
+def prepack_weight_if_needed(weight):
+    if not cpu_has_amx_support() or weight.device != torch.device("cpu"):
```

Comment:
```python
if weight.device != torch.device("cpu"):
    return weight
if not cpu_has_amx_support():
    return weight

return convert_weight_packed(weight)
```
```

Reply:
Changed in https://github.com/mingfeima/sglang/pull/8/commits/24fe31bccfd438ac69737b9d3f9cdef74cfc9ab6